src/js/model/string/test-string-model.html

```diff
@@ -0,0 +1,46 @@
+<html>
+	<head>
+		<meta charset="utf-8">
+		<script src="../../../../bower_components/web-component-tester/browser.js"></script>
+		<link rel="import" href="../../../../bower_components/webcomponentsjs/webcomponents-lite.js">
+
+		<script src="string.js"></script>
+	</head>
+	<body>
+		<script>
+
+suite("String model constructor", function()
+{
+	var stringHex;
+	var returnedObj;
+
+	suiteSetup(function()
+	{
+		stringhex = "1ABCDE00FGHI";
+		returnedObj = AMPParse.buildString(stringhex);
+	});
+
+	test("returns the correct value", function()
+	{
+		expect(returnedObj.returnValue.value).to.be.a("string").and.equal("1ABCDE");
```

Comment:
The string you have here, 1A, BC, DE should be the string [SUB], (not in ascii), (not in ascii). Remember, each byte value gets converted into an ASCII character.